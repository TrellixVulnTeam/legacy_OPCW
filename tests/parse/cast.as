# Explicit "safe" casting is done through the 'as' operator.
10 as uint;
2 as int;
0x0A as char;
# "A" as str;
