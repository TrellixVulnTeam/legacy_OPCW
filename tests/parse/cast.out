ModuleDecl <?> _
  Nodes <?> 
    CastExpr <?>
      IntegerExpr <?> 10 (10)
      Ident <?> uint
    CastExpr <?>
      IntegerExpr <?> 2 (10)
      Ident <?> int
    CastExpr <?>
      IntegerExpr <?> 0A (16)
      Ident <?> char
